api/query/cache/index/index.go

```diff
@@ -34,6 +34,7 @@ var (
 	errSomeShardsRequired = errors.New("Index must have at least one shard")
 	errUnexpectedRuns     = errors.New("Unexpected number of runs")
 	errZeroRun            = errors.New("Cannot ingest run with ID of 0")
+	errEmptyReport        = errors.New("Report contains no results")
```

Comment:
Drive-by optional nit: the convention is to not capitalize the error strings (they are usually logged in formats like "Failed to ...: [error string]").

Reply:
Maybe in another PR.